dsgrid/config/dimension_mapping_base.py

```diff
@@ -6,17 +6,66 @@
 from semver import VersionInfo
 
 from .dimensions import DimensionReferenceModel
-from dsgrid.data_models import DSGBaseModel
+from dsgrid.data_models import DSGBaseModel, DSGEnum
 from dsgrid.dimension.base_models import DimensionType
 from dsgrid.utils.versioning import handle_version_or_str
+from dsgrid.exceptions import DSGInvalidDimensionMapping
 
 
 logger = logging.getLogger(__name__)
 
 
+class DimensionMappingType(DSGEnum):
+    # default from_fraction col, FRACTION_SUM_EQ1
+    ONE_TO_ONE = "one_to_one"  # includes rename, down-selection
+    MANY_TO_ONE_AGGREGATION = "many_to_one_aggregation"
+
+    # default from_fraction col, FRACTION_SUM_NOT_EQ1
+    DUPLICATION = "duplication"
+
+    # input from_fraction col, FRACTION_SUM_EQ1
+    MANY_TO_MANY_AGGREGATION = "many_to_many_aggregation"
+    ONE_TO_MANY_DISAGGREGATION = "one_to_many_disaggregation"
+    MANY_TO_MANY_DISAGGREGATION = "many_to_many_disaggregation"
+
+    # input from_fraction col, FRACTION_SUM_NOT_EQ1
+    ONE_TO_ONE_MULTIPLICATION = "one_to_one_multiplication"
+    ONE_TO_MANY_MULTIPLICATION = "one_to_many_multiplication"
+    MANY_TO_ONE_MULTIPLICATION = "many_to_one_multiplication"
+    MANY_TO_MANY_MULTIPLICATION = "many_to_many_multiplication"
```

Comment:
We might need this for scaling up ComStock results so that the floor area matches floor area from AEO. Recall ComStock only models a subset of all comm. building types. We can do this scaling in the form of a dim mapping or we do this directly in the ComStock data so the data submitted to project is already scaled.